erpnext/stock/stock_ledger.py

```diff
@@ -867,3 +870,8 @@ def _round_off_if_near_zero(number: float, precision: int = 6) -> float:
 		return 0
 
 	return flt(number)
+
+def _switch_values(target, keys, value_from=None, value_to=""):
```

Comment:
@18alantom Very bad choice of function naming, variable naming and implementation, the name does not tell what it does.

It should be like: 
```suggestion
def sanitize_args(args, keys):
	for k in keys:
		if args[k] == None:
			args[k] == ''
	return args
```

Reply:
@nabinhait I agree; should the `sanitize_args` be in stock_ledger.py, or moved elsewhere?

Reply:
keep it here only